### DISCUSSION TITLE
Question for adding plugin to the "JBrowse 2 Plugin Store"

### DISCUSSION THREAD

**Discussion by Chi-HsienChang** [GMOD/jbrowse-components#3656] [Q&A] [answered]
Hi JBrowse2 developers,

Could I apply to add the plugin to the "JBrowse 2 Plugin Store"?

The plugin is: https://github.com/NAL-i5K/jbrowse-plugin-linkout
 
![image](https://user-images.githubusercontent.com/114167559/233435984-3b77dbd3-20ba-470f-a33d-8275efc1922b.png)

Thank you so much!

**Comment by garrettjstevens** [comment, marked as answer]
Hi @Chi-HsienChang, we'd be happy to take a look at your plugin. To apply, you'll need to open a PR in this repository: https://github.com/GMOD/jbrowse-plugin-list. There are instructions for what to include in the README there. Let us know if you have additional questions.

**Reply by Chi-HsienChang** [reply]
Hi @garrettjstevens,

I understand it, I open a PR in this repository, thank you so much!

**Reply by Chi-HsienChang** [reply]
Hi @garrettjstevens,

![image](https://user-images.githubusercontent.com/114167559/233803898-4d91ba71-e7b1-4bee-ab8b-e0c193e0534a.png)

I open a PR in this repository, can you help me add to plugin store? Thank you so much!
